solr/core/src/java/org/apache/solr/core/HdfsDirectoryFactory.java

```diff
@@ -452,8 +452,6 @@ public long size(String path) throws IOException {
     } catch (IOException e) {
       log.error("Error checking if hdfs path exists", e);
       throw new SolrException(ErrorCode.SERVER_ERROR, "Error checking if hdfs path exists", e);
-    } finally {
-      IOUtils.closeQuietly(fileSystem);
```

Comment:
Why do we no longer need to close this?

Reply:
Without removing this I got filesystem already closed exceptions.

We are getting the fileSystem using `FileSystem fileSystem = getCachedFileSystem(path)`. And cache will take care of the close as stated here: https://github.com/apache/solr/blob/main/solr/core/src/java/org/apache/solr/core/HdfsDirectoryFactory.java#L462

```
// no need to close the fs, the cache will do it
```